lib/y-richtext.coffee

```diff
@@ -1,4 +1,8 @@
 BaseClass = (require "./misc.coffee").BaseClass
+Y = require '../../yjs/lib/y.coffee'
```

Comment:
no! as I said before, you must not include Y here! Y must be included by the user. If you do it like this, you'll almost definitely run into weird problems.

_getModel(Y,Operation) is supposed to serve Y!